cmd/zap/cmd/root.go

```diff
@@ -24,6 +24,11 @@ import (
 
 var segment *zap.Segment
 
+const (
+	sectionInvertedTextIndex = iota
+	sectionFaissVectorIndex
+)
```

Comment:
We don't need to redeclare these here, use zap.sectionInvertedTextIndex and zap.sectionFaissVectorIndex

Reply:
so i'd have to export those two vars in section.go. i've made the change, please let me know that's fine.

Reply:
Ya export away.